src/main/java/com/training/web/pages/rccl/LoginPage.java

```diff
@@ -0,0 +1,33 @@
+package com.training.web.pages.rccl;
+
+import com.codeborne.selenide.Condition;
+
+import static com.codeborne.selenide.Selenide.$x;
+
+public class LoginPage {
+
+    private String email = "//input[@aria-label='Email address or username']";
+    private String password = "//input[@aria-label='Password']";
+    private String signIn = "//button[text()=' Sign in ']";
+
+    public LoginPage enterEmail(String emailId) {
+        $x(email).shouldBe(Condition.visible).sendKeys(emailId);
+        return this;
+    }
+
+    public LoginPage enterPassword(String passwordText) {
+        $x(password).shouldBe(Condition.visible).sendKeys(passwordText);
+        return this;
+    }
+
+    public LoginPage clickSignIn() {
```

Comment:
shouldn't this return HomePage?

Reply:
@akhil7000 click signin is inside for loop. so i had removed initializing homePage inside this method since it was needed only on the 10th attempt